packages/embeds/embed-core/src/embed.test.ts

```diff
@@ -115,6 +116,17 @@ describe("Cal", () => {
         expect(iframe.src).toContain("email=test%40example.com");
       });
 
+      it("should allow configuring reserved params through direct URL params to embed calLink(as it is explicitly passed by user)", () => {
```

Comment:
Added one more scenario that people might be using embed in. By directly passing query params in calLink